### PR TITLE
(feat) O3-3156: Submitting the start visit form should mutate appointments

### PR DESCRIPTION
**Pull request by mogoodrich**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing.en-US#contributing-guidelines) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://om.rs/o3ui).
- [ ] My work includes tests or is validated by existing tests.

## Summary

This PR makes it so that submitting the Start Visit form also mutates the Appointments endpoint so any appointment statuses updated via the form get propagated across the app. 

## Related Issue

https://openmrs.atlassian.net/browse/O3-3156

### CODE REVIEW

File: packages/esm-patient-chart-app/src/visit/hooks/useMutateAppointments.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { useSWRConfig } from 'swr';
+import { useCallback } from 'react';
+
+// this is copied directly from a similar hook in the appointments-app in patient management; ideally at some point we could import that hook directly
+const appointmentUrlMatcher = '/ws/rest/v1/appointment';
```
Comment (denniskigen) [inline packages/esm-patient-chart-app/src/visit/hooks/useMutateAppointments.tsx:5]: Minor detail:

```ts
import { restBaseUrl } from '@openmrs/esm-framework';
```

```suggestion
const appointmentUrlMatcher = '${restBaseUrl}/appointment';
```

**Review by denniskigen — APPROVED** [review]
LGTM, @mogoodrich. One suggestion about `restBaseUrl`.

**Review by mseaton — APPROVED** [review]

**Comment by denniskigen** [comment]
@mogoodrich you'll need to grab that import from the framework

**Comment by ojwanganto** [comment]
Looks good once you refine the import bit.

**Review by ojwanganto — APPROVED** [review]

**Pull request merged**